site/content/migrating/01-migrating.md

```diff
@@ -0,0 +1,367 @@
+---
+title: Migrating
+---
+
+
+Until we reach version 1.0, there may be occasional changes to the project structure Sapper expects.
+
+
+### 0.25 to 0.26
+
+The most significant change yet: Sapper is now built on Svelte 3.
+
+#### Importing Sapper
+
+Your app is now built to `src/node_modules/@sapper` — this allows you to easily import it from anywhere in your source code. Update your `server.js`...
```

Comment:
I dunno if I'd say that your _app_ is built to there - your app is still built to `__sapper__/dev` and `__sapper__/build` - It's stuff that you want to import that's built to `src/node_modules/@sapper` before your app's bundled. I don't know how important of a thing to be super precise about this is.

Reply:
Gah, you're right. Honestly I was thinking about turning this whole page into something more like the docs page on svelte.dev and introducing some kind of a tutorial, since this page is kind of trying to do what the old `/guide` did.

Reply:
This _is_ the migration page so I don't think a more digestible or lengthy version of this is really necessary - but it might be nice to have separate tutorial and docs pages sometime in the future. Although I have no idea what implementing a Sapper REPL would involve, and we shouldn't hold up other things for that.

Reply:
Tweaked it slightly — still not perfect, but happy enough with it for now